src/components/Map/GroupMap.vue

```diff
@@ -103,23 +128,34 @@ export default {
     usersWithLocation () {
       return this.users.filter(hasLocation)
     },
-    selectedMarkerIds () {
-      let ids = []
-      if (this.selectedStoreId) {
-        ids.push(this.storeMarkerId(this.selectedStoreId))
+    groupsWithLocation () {
+      return (this.groups && this.groups.filter(hasLocation)) || []
+    },
+    selectedMarkers () {
+      if (this.selectedStore) {
+        const markers = []
+        if (hasLocation(this.selectedStore)) {
+          markers.push(storeMarker(this.selectedStore))
+        }
         if (this.showUsers) {
-          ids.push(...this.usersWithLocation.map(user => user.id).map(this.userMarkerId))
+          markers.push(...this.usersWithLocation.map(userMarker))
```

Comment:
This is a duplication of marker creation as another one is created below in the `markers()` function. It would be unclear which is actually used without looking how they are used in the map. In this case it just pulls the marker id out later on.

An improvement might be to have a `selected` property on the marker, and remove the `selected-markers` prop to the map.